docs/_chapters/150-build.md

```diff
@@ -50,8 +50,9 @@ For example, the Maven plugin that is built-in to bndlib has an extension file c
 	# Plugin maven setup
 	#
 	-plugin.maven = aQute.bnd.plugin.maven.MavenPlugin
-	
-	
+
+
+​	
```

Comment:
`  `some whitespaces here.  this block must not change

Reply:
hope this fixes this